### PR TITLE
Fix try-with-resources warnings

### PR DESCRIPTION
**Pull request by edewata**
This PR fixes some `try-with-resources` warnings reported by SonarCloud:
https://sonarcloud.io/project/issues?resolved=false&severities=BLOCKER&types=BUG&id=dogtagpki_pki&open=AYE5lMP2tLTZxWLwchRt

### CODE REVIEW

File: base/server/src/main/java/com/netscape/cms/servlet/common/CMSTemplate.java [hunk 1]
```diff
@@ -288,23 +290,16 @@ private String loadFile(File template) throws IOException {
...
-        while ((line = in.readLine()) != null) {
-            buf.append(line);
-            buf.append('\n');
-        }
-        try {
-            in.close();
-            inStream.close();
-        } catch (IOException e) {
-            logger.warn("CMSTemplate: " + CMS.getLogMessage("CMSGW_ERR_CLOSE_TEMPL_FILE", mAbsPath, e.getMessage()), e);
+        try (InputStream inStream = new FileInputStream(template);
```
Comment (ckelleyRH) [inline base/server/src/main/java/com/netscape/cms/servlet/common/CMSTemplate.java:294]: o_O Lots going on in this one!

**Review by ckelleyRH — APPROVED** [review]
LGTM, thanks!

**Review by fmarco76 — APPROVED** [review]
LGTM

**Comment by edewata** [comment]
@ckelleyRH @fmarco76 Thanks!

**Pull request merged**